### ISSUE TITLE
[X-Fi] Audio pops/clicks in Half-Life

### ISSUE THREAD

**Issue by Kappa971** [LAGonauta/MetaAudio#48]
Hi @LAGonauta, we already talked about it in another discussion.
You can easily notice pops or clicks in the sounds in the training level in Half-Life using a Sound Blaster X-Fi.
Currently it is possible to solve this problem by using OpenAL Soft instead of the Creative driver.

I create this as a reminder for a possible fix in the future.

**Comment by Kappa971** [comment]
I tried Half Life WON version (CD) with Creative Alchemy. I had to lower the volume of the sounds to 0.50, for the usual clipping problem (but I also had to do this with Half Life Steam and MetaAudio. With the X-Fi, over the months, I understood that the developers in the days of DS3D/OpenAL didn't bother to adjust the volume of the sounds correctly, so it's not surprising). 
Anyway, with Half Life Steam+MetaAudio, the sound seems more muffled than in Half Life WON+ALchemy, also in Half Life WON I don't notice clicks or pops.

**Comment by whyronimus** [comment]
@Kappa971 how did you get MetaAudio working with WON Half-Life?
I get an error / crash with "could not load filesystem.dll". 
Any help appreciated, TIA

**Comment by Kappa971** [comment]
> @Kappa971 how did you get MetaAudio working with WON Half-Life? I get an error / crash with "could not load filesystem.dll". Any help appreciated, TIA

Non-Steam version of Half Life supports DirectSound3D and EAX, MetaAudio is for Half Life Steam.
